### ISSUE TITLE
Failing to access the admin panel

### ISSUE THREAD

**Issue by carloswallaceratibu** [scriptandtools/Real-Estate-website-in-PHP#1]
Goodday how are you l am failing to access the admin panel

**Comment by carloswallaceratibu** [comment]
user name and password not correct

**Comment by toby11** [comment]
yes I had same problem - this seems to work, I would reset all passwords once your in.
username : admin
password : codeastro.com

there seems to be no way to change admin password - this project seems half finished.

**Comment by sagarxrijal21** [comment]
can u help me. the login id and pw are wrong
